### PR TITLE
Added module build

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

**Comment by alexreardon** [comment]
Thanks for this - it is looking good. I'll test it out on my machine

File: .babelrc.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { BABEL_ENV, NODE_ENV } = process.env;
+const modules = BABEL_ENV === 'cjs' || NODE_ENV === 'test' ? 'commonjs' : false;
```
Comment (alexreardon) [inline .babelrc.js]: is this what the cross-env is for?

Comment (Andarist) [reply .babelrc.js]: `cross-env` just sets any env variables consistently on all platforms (so it should work also on windows)

**Comment by alexreardon** [comment]
This is looking good.

Can you please add the following browserlist?

```
ie >= 11
last 1 Edge version
last 1 Firefox version
last 1 Chrome version
last 1 Safari version
last 1 iOS version
last 1 Android version
last 1 ChromeAndroid version
```

**Comment by Andarist** [comment]
@alexreardon done, although it could probably be shortened to:
```
{ browsers: ['ie >= 11', 'last 1 version']}
```

**Pull request merged**